### PR TITLE
Issue comp 16

### PR DESCRIPTION
**Pull request by balmas**
for alpheios-project/components#16

see PR alpheios-project/components#740 for discussion

### CODE REVIEW

File: src/controllers/wordlist-controller.js [hunk 1]
```diff
@@ -152,13 +152,13 @@ export default class WordlistController {
   * Emits a WORDITEM_UPDATED event
   */
   onDefinitionsReady (data) {
-    let wordItem = this.getWordListItem(data.homonym.language,data.homonym.targetWord)
+    let wordItem = this.getWordListItem(LanguageModelFactory.getLanguageCodeFromId(data.homonym.languageID),data.homonym.targetWord)
```
Comment (irina060981) [inline src/controllers/wordlist-controller.js:155]: Bridget, the same question here - why did you remove language property?

**Review by irina060981 — APPROVED** [review]
Looks ok!

**Review by kirlat — COMMENTED** [review]
I have the same question for discussion here: whether would we like to add a standard Alpheios prefix to all Alpheios-originated messages

**Review by kirlat — APPROVED** [review]

**Comment by balmas** [comment]
> I have the same question for discussion here: whether would we like to add a standard Alpheios prefix to all Alpheios-originated messages

done.
